.github/workflows/docscheck.yml

```diff
@@ -50,20 +51,20 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-check
           textlint_flags: "{README.md,version-history.md,docs/**}"
+      - name: Run linkspector to check links
+        uses: umbrelladocs/action-linkspector@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
So the letter case for GitHub env doesn't matter if it works now. I didn't know that.

Reply:
Not sure what the situation is, I changed all to upper because it makes more sense to be consistent. I know that GITHUB_TOKEN is optional for this one. But weirdly their doc suggests the lower-case in https://github.com/UmbrellaDocs/action-linkspector?tab=readme-ov-file#github_token